### DISCUSSION TITLE
Issue when reset_password_command used with non id 1 admin account

### DISCUSSION THREAD

**Discussion by babs** [grafana/grafana#37256] [General]
**What happened**:

In https://github.com/grafana/grafana/blob/master/pkg/cmd/grafana-cli/commands/reset_password_command.go : assumption be is made that first account is id 1 and breaks reset-password-command if not 1.

It happens when using clustered database servers like mysql multimaster or galera

**What you expected to happen**:

Use the admin username instead of assuming it's id is 1.

**How to reproduce it (as minimally and precisely as possible)**:

Configure a mysql server with my.cnf

```
[mysqld]
auto_increment_increment = 2
auto_increment_offset  = 1
```

Start grafana against it, try to reset the password.

You can edit the row, change the is back to 1, try the reset, it works.

**Anything else we need to know?**:

**Environment**:
- Grafana version: v6.1.0 (03346b6)
- Data source type & version: mysql 
- OS Grafana is installed on: Debian
- User OS & Browser: N/A
- Grafana plugins: None
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
